src/pages/home/report/ReportActionsList.tsx

```diff
@@ -341,13 +355,20 @@ function ReportActionsList({
 
     const scrollToBottomForCurrentUserAction = useCallback(
         (isFromCurrentUser: boolean) => {
-            // If a new comment is added and it's from the current user scroll to the bottom otherwise leave the user positioned where
-            // they are now in the list.
+            // If a new action is added and it's from the current user scroll to the bottom,
+            // otherwise leave the user positioned where they are now in the list.
             if (!isFromCurrentUser || !hasNewestReportActionRef.current) {
                 return;
             }
-            InteractionManager.runAfterInteractions(() => reportScrollManager.scrollToBottom());
+            InteractionManager.runAfterInteractions(() => {
+                // Reset highlight if necessary
+                if (linkedReportActionID) {
+                    Navigation.setParams({reportActionID: ''});
+                }
+                reportScrollManager.scrollToBottom();
+            });
```

Comment:
@sobitneupane: If `!hasNewestReportActionRef.current` we don't reset the highlight or scroll to the bottom.
In a chat with many new messages, when the current user creates a new action, the scroll doesn't go to the bottom, consequently the yellow highlight isn't cleared.\
Should we leave it like that, or should we clear the yellow highlight regardless of `hasNewestReportActionRef.current`? What do you think?

Reply:
@brunovjk I believe we should clear the highlight if any new action is taken by the user.

Reply:
> @brunovjk I believe we should clear the highlight if any new action is taken by the user.

Exactly, I tested calling `clearHighlight` as soon as the user performs an action, but the application looks strange and renders more than necessary, blinking you know.

I tried several different flows/steps to call `clearHighlight`, the best way I found was to call it in `runAfterInteractions` here:
https://github.com/Expensify/App/blob/896c14c96333c34a43a82191a7170333dddd49cf/src/pages/home/report/ReportActionsList.tsx#L349

but for it to work well, everything before, like this check:
https://github.com/Expensify/App/blob/896c14c96333c34a43a82191a7170333dddd49cf/src/pages/home/report/ReportActionsList.tsx#L346
should work too. I believe they are trying to resolve it as we speak: https://github.com/Expensify/App/pull/39951#discussion_r1647084530